neo/io/neomatlabio.py

```diff
@@ -199,131 +243,146 @@ def __init__(self, filename=None):
         """
         import scipy
 
-        if Version(scipy.version.version) < Version('0.12.0'):
-            raise ImportError("your scipy version is too old to support "
-                                    + "MatlabIO, you need at least 0.12.0. "
-                                    + "You have %s" % scipy.version.version)
+        if Version(scipy.version.version) < Version("0.12.0"):
+            raise ImportError(
+                "your scipy version is too old to support "
+                + "MatlabIO, you need at least 0.12.0. "
+                + "You have %s" % scipy.version.version
```

Comment:
```suggestion
                f"your scipy version is too old to support "
                + f"MatlabIO, you need at least 0.12.0. "
                + f"You have {scipy.version.version}" 
```

I'm drafting a PR for after the release to convert everything to f-strings.

Reply:
good idea.